ansible/roles/ocp4-workload-ccnrd/tasks/install-mta.yaml

```diff
@@ -1,31 +1,31 @@
 ---
-# deploy rhamt
-- name: Deploy RHAMT
+# deploy mta
+- name: Deploy MTA
   shell: oc process -f - | oc create -n labs-infra  -f -
   args:
-    stdin: "{{ lookup('file', './files/rhamt-template.yaml') }}"
+    stdin: "{{ lookup('file', './files/mta-template.yaml') }}"
 
-- name: TODO - scale RHAMT executor to quarter the number of users
+- name: TODO - scale MTA executor to quarter the number of users
```

Comment:
Is this still TODO? Please update task name if not.

Reply:
Removed.